cudaaligner/src/alignment.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include <claragenomics/cudaaligner/alignment.hpp>
+
+#include <iostream>
+
+namespace claragenomics
+{
+
+namespace cudaaligner
+{
+
+std::ostream& operator<<(std::ostream& os, const FormattedAlignment& formatted_alignment)
+{
+    std::size_t line_length              = (formatted_alignment.linebreak_after == 0) ? formatted_alignment.query.size() : formatted_alignment.linebreak_after;
+    std::size_t number_of_complete_lines = (std::size_t)(formatted_alignment.query.size() / line_length);
+    for (std::size_t i = 0; i < number_of_complete_lines; i++)
+    {
+        std::size_t substr_start_idx = i * line_length;
+        os << formatted_alignment.query.substr(substr_start_idx, line_length) << '\n'
+           << formatted_alignment.pairing.substr(substr_start_idx, line_length) << '\n'
+           << formatted_alignment.target.substr(substr_start_idx, line_length) << '\n';
+    }
+    std::size_t last_line_length = formatted_alignment.query.size() - number_of_complete_lines * line_length;
+    if (last_line_length != 0)
+    {
+        std::size_t last_line_start_idx = number_of_complete_lines * line_length;
+        os << formatted_alignment.query.substr(last_line_start_idx, last_line_length) << '\n'
+           << formatted_alignment.pairing.substr(last_line_start_idx, last_line_length) << '\n'
+           << formatted_alignment.target.substr(last_line_start_idx, last_line_length) << '\n';
+    }
+    os << std::endl;
+    return os;
+}
+
+} // namespace cudaaligner
+} // namespace claragenomics
```

Comment:
add a newline after `// namespace claragenomics` (I'm astonished that clang-format doesn't take care of that...)

Reply:
There doesn't seem to be such an option.
Surprisingly, gcc doesn't throw a warning since C++11 as well.
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=68994